editoast/src/views/infra/auto_fixes.rs

```diff
@@ -602,194 +601,165 @@ mod test {
-        })));
-        assert!(operations.contains(&Operation::Delete(DeleteOperation {
-            obj_id: speed_section.get_id().to_string(),
-            obj_type: ObjectType::SpeedSection,
-        })));
+
+        for obj in [&catenary, &operational_point, &speed_section] {
+            assert!(operations.contains(&Operation::Delete(DeleteOperation {
+                obj_id: obj.get_id().to_string(),
+                obj_type: obj.get_type(),
+            })))
+        }
     }
 
     #[rstest::rstest]
     async fn out_of_range_must_be_ignored() {
         let app = create_test_service().await;
-        let small_infra = small_infra(db_pool()).await;
-        let small_infra_id = small_infra.id();
+        let empty_infra = empty_infra(db_pool()).await;
+        let empty_infra_id = empty_infra.id();
+
+        let track: RailjsonObject = TrackSection {
+            id: "test_track".into(),
+            length: 1_000.0,
+            ..Default::default()
+        }
+        .into();
 
         let catenary: RailjsonObject = Catenary {
             track_ranges: vec![ApplicableDirectionsTrackRange {
-                begin: 100000000000.0,
-                end: 100000000001.0,
+                track: "test_track".into(),
+                begin: 250.0,
+                end: 1250.0,
                 ..Default::default()
             }],
             ..Default::default()
         }
         .into();
-        let req_create = get_create_operation_request(catenary.clone(), small_infra_id);
-        assert_eq!(
-            call_service(&app, req_create).await.status(),
-            StatusCode::OK
-        );
 
         let operational_point: RailjsonObject = OperationalPoint {
             parts: vec![OperationalPointPart {
-                position: 10000000000000.0,
+                track: "test_track".into(),
+                position: 1250.0,
                 ..Default::default()
             }],
             ..Default::default()
         }
         .into();
-        let req_create = get_create_operation_request(operational_point.clone(), small_infra_id);
-        assert_eq!(
-            call_service(&app, req_create).await.status(),
-            StatusCode::OK
-        );
 
         let speed_section: RailjsonObject = SpeedSection {
             track_ranges: vec![ApplicableDirectionsTrackRange {
-                begin: 100000000000.0,
-                end: 100000000001.0,
+                track: "test_track".into(),
+                begin: 250.0,
+                end: 1250.0,
                 ..Default::default()
             }],
             ..Default::default()
         }
         .into();
-        let req_create = get_create_operation_request(speed_section.clone(), small_infra_id);
-        assert_eq!(
-            call_service(&app, req_create).await.status(),
-            StatusCode::OK
-        );
 
-        let track_section: RailjsonObject = TrackSection {
-            slopes: vec![Slope {
```

Comment:
To match previous test, you may add a slope to the track and test it doesn't get deleted, no?
The idea was to prevent from implem of https://github.com/osrd-project/osrd/issues/5863 leaking on https://github.com/osrd-project/osrd/issues/5943

Reply:
Slopes are not individual objects but just part of the definition of a track section. So just checking the TS isn't deleted is enough (if I understand your comment correctly).

Reply:
Yes, and we can get an out of range on track section itself for its slope.
So yes, adding an out-of-range slope and checking the TS isn't proposed for deletion.

Reply:
> and test it doesn't get deleted

Oh I thought you meant the slope itself. Then yes, it's definitely a good idea :)

Reply:
Yeah, This is the only part of the test I did not adapt, the reason is that I encounter an issue with slopes in this test (the test was not failing, but was not good either). I removed this intentionally, until I determine if there is a real bug. i'm currently investigating it, so i'll quickly add the test with slopes out of range in this PR if I found nothing.